pkg/sloop/webserver/webfiles/filter.js

```diff
@@ -108,6 +108,10 @@ function setFiltersAndReturnQueryUrl(defaultLookback, defaultKind, defaultNamesp
         var now =  new Date().toISOString()
         selectedEndTime = now.substring(0, now.length-5)
     }
+    else{
+        var endTime = new Date(selectedEndTime);
+        selectedEndTime = endTime.toISOString().slice(0,-1)
```

Comment:
please make sure to test a bunch of use cases to make sure there is no regression

Reply:
good idea, i added two cases for end date in time range test